server/datastore/mysql/migrations/tables/20240129162819_AddPrefixToWindowsUpdatesProfiles.go

```diff
@@ -0,0 +1,44 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20240129162819, Down_20240129162819)
+}
+
+func Up_20240129162819(tx *sql.Tx) error {
+	stmt := `
+		UPDATE
+			mdm_windows_configuration_profiles
+		SET
+			profile_uuid = CONCAT("w", profile_uuid)
```

Comment:
This will auto-update the `updated_at` timestamp column and that messes up profiles delivery (an issue I created in a past migration :( ). To ensure this is not the case, you need to explicitly set `updated_at` to its current value:
```suggestion
		UPDATE
			mdm_windows_configuration_profiles mwcp
		SET
			profile_uuid = CONCAT("w", profile_uuid),
			updated_at = mwcp.updated_at
```